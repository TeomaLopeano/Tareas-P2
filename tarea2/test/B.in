cantidadColCadenas 0
insertarEnColCadenas 3 3.0 9
estaEnColCadenas 3 1
cadenaDeColCadenas 1
estaEnColCadenas 3 9
insertarEnColCadenas 5 5.0 9
insertarEnColCadenas 6 6.0 9
insertarEnColCadenas 7 7.0 9
cantidadColCadenas 9
estaEnColCadenas 7 9
estaEnColCadenas 5 9
infoEnColCadenas 7 9
infoEnColCadenas 5 9
cadenaDeColCadenas 9
removerDeColCadenas 6 9
removerDeColCadenas 7 9
removerDeColCadenas 3 9
cadenaDeColCadenas 9
Fin