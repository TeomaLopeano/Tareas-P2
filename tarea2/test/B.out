1>La cantidad en la cadena 0 es 0.
2>Se insertó (3,3.00) en la cadena 9.
3>3 no está en la cadena 1.
4>La cadena 1 es :
5>3 está en la cadena 9.
6>Se insertó (5,5.00) en la cadena 9.
7>Se insertó (6,6.00) en la cadena 9.
8>Se insertó (7,7.00) en la cadena 9.
9>La cantidad en la cadena 9 es 4.
10>7 está en la cadena 9.
11>5 está en la cadena 9.
12>La primera aparición de 7 en la cadena 9 es (7,7.00)
13>La primera aparición de 5 en la cadena 9 es (5,5.00)
14>La cadena 9 es :(7,7.00)(6,6.00)(5,5.00)(3,3.00)
15>Se removió la primera aparición de 6 de la cadena 9.
16>Se removió la primera aparición de 7 de la cadena 9.
17>Se removió la primera aparición de 3 de la cadena 9.
18>La cadena 9 es :(5,5.00)
19>Fin.
